src/pages/demo/personProfiles/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { drupalClient } from '@/utils/drupalClient'
+import { PersonProfile } from '@/components/node/person_profile'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import Container from '@/components/container'
+
+const PersonProfilePage = ({ nodes }) => {
+  if (!nodes) return
+  return (
+    <>
+      <Container className="container">
+        <div id="content" className="interior">
+          <main className="va-l-detail-page va-facility-page">
+            <div className="usa-grid usa-grid-full">
+              <div className="usa-width-three-fourths">
+                {nodes.map((node, index) => (
+                  <div key={index}>
+                    <PersonProfile node={node} />
+                  </div>
+                ))}
+              </div>
+            </div>
+          </main>
+        </div>
+      </Container>
+    </>
+  )
+}
+
+export default PersonProfilePage
+
+export async function getStaticProps(context) {
+  const params = new DrupalJsonApiParams()
+  params
+    .addFilter('status', '1')
+    .addFilter('field_office', null, 'IS NOT NULL')
+    .addFilter('field_media', null, 'IS NOT NULL')
+    .addInclude(['field_office', 'field_media'])
```

Comment:
It may be good for us to start putting limits on how many objects we retrieve for these demo pages. We can add this with `addPage()`, i.e. `.addPage(10)` to limit ourselves to 10 objects.